data/sql/updates/pending_db_world/rev_1550579754213080400.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1550579754213080400');
+UPDATE `creature_template` SET `speed_walk` = 1, `speed_run` = 1 WHERE `entry` IN (27755, 27756, 27692);
```

Comment:
this will set flying speed to 1, could you confirm that flying speed needs to be 1 instead of 3

https://github.com/azerothcore/azerothcore-wotlk/blob/3e1ed410d035ff045247af0db17c062708d4e442/src/server/scripts/Northrend/Nexus/Oculus/oculus.cpp#L290

Reply:
yea. flight must be 1, otherwise dragons fly slowly

Reply:
that's weird, with 1 are slower than 3 (actual value)? Is there any code making them slower for higher numbers?

Reply:
I believe it is having some conflict (MOVE_RUN)